src/main/java/frc/robot/subsystems/IntakeSubsystem.java

```diff
@@ -4,36 +4,84 @@
 
 package frc.robot.subsystems;
 
+import static edu.wpi.first.units.Units.Degrees;
 import static frc.robot.Constants.IntakeConstants.*;
 
+import com.ctre.phoenix6.hardware.TalonFX;
 import com.revrobotics.spark.SparkLowLevel.MotorType;
 import com.revrobotics.spark.SparkMax;
+import edu.wpi.first.units.measure.Angle;
 import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
 import edu.wpi.first.wpilibj2.command.Command;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 import frc.robot.Constants.MotorIDConstants;
+import utilities.HowdyPID;
 
 public class IntakeSubsystem extends SubsystemBase {
   /** Creates a new IntakeSubsystem. */
   private SparkMax intakeMotor;
 
+  private TalonFX pivotMotor;
+  private SparkMax conveyorMotor;
+  private HowdyPID pivotPID;
+  private Angle pivotSetpoint = kPivotRetractAngle;
+
   public IntakeSubsystem() {
-    intakeMotor = new SparkMax(MotorIDConstants.kIntakeMotor, MotorType.kBrushed);
+    intakeMotor = new SparkMax(MotorIDConstants.kIntakeMotor, MotorType.kBrushless);
+    pivotMotor = new TalonFX(MotorIDConstants.kPivotMotor);
+    conveyorMotor = new SparkMax(MotorIDConstants.kConveyorMotor, MotorType.kBrushless);
+    pivotPID = new HowdyPID(kPivotP, kPivotI, kPivotD);
+  }
+
+  private void setPivotMotor(double speed) {
+    pivotMotor.set(speed);
+  }
+
+  private void setConveyerMotor(double speed) {
+    conveyorMotor.set(speed);
+  }
+
+  private void setIntakeMotor(double speed) {
+    intakeMotor.set(speed);
+  }
+
+  public Angle getPivotPositionCommand() {
+    return pivotMotor.getPosition().getValue();
+  }
+
+  public Command extendPivotCommand() {
+    return runOnce(() -> pivotSetpoint = kPivotExtendAngle);
+  }
+
+  public Command retractPivotCommand() {
+    return runOnce(() -> pivotSetpoint = kPivotRetractAngle);
   }
 
   // Made a command to spin clockwise
   public Command IntakeCommand() {
-    return startEnd(() -> intakeMotor.set(kSpeed), () -> intakeMotor.set(0));
+    return startEnd(() -> setIntakeMotor(kIntakeSpeed), () -> setIntakeMotor(0));
   }
 
   // Made a command to spin counter clockwise
   public Command OuttakeCommand() {
-    return startEnd(() -> intakeMotor.set(-kSpeed), () -> intakeMotor.set(0));
+    return startEnd(() -> setIntakeMotor(-kIntakeSpeed), () -> setIntakeMotor(0));
+  }
+  public Command ConveyorCommand() {
+    return startEnd(() -> setConveyerMotor(kConveyorSpeed), () -> setConveyerMotor(0));
+  }
+  public Command ReverseConveyorCommand() {
+    return startEnd(() -> setConveyerMotor(-kConveyorSpeed), () -> setConveyerMotor(0));
+  }
+  public double calculatePivotPID(double setpoint) {
+    return pivotPID
+        .getPIDController()
+        .calculate(pivotMotor.getPosition().getValueAsDouble(), setpoint);
   }
 
   @Override
   public void periodic() {
     // This method will be called once per scheduler run
+    setPivotMotor(calculatePivotPID(pivotSetpoint.in(Degrees)));
```

Comment:
There is a unit mismatch here. `getPosition()` returns a position in Revolutions of the motor, not Degrees. So the setpoint also needs to be in Revolutions. 

Additionally, make sure you account for any gear reductions (Check with CAD for this).